js/views/modals/Settings/Page.js

```diff
@@ -21,6 +22,11 @@ export default class extends baseVw {
 
     this.profile = app.profile.clone();
     this.listenTo(this.profile, 'sync', () => app.profile.set(this.profile.toJSON()));
+
+    this.socialAccounts = this.createChild(SocialAccounts, {
+      collection: this.profile.get('contactInfo').get('social').clone(),
```

Comment:
You don't need to clone here. The profile model is already clones (including nested objects) a couple lines above.